models/lfs.go

```diff
@@ -0,0 +1,98 @@
+package models
+
+import (
+	"errors"
+	"github.com/go-xorm/xorm"
+	"time"
+)
+
+type LFSMetaObject struct {
+	ID           int64     `xorm:"pk autoincr"`
+	Oid          string    `xorm:"INDEX NOT NULL"`
+	Size         int64     `xorm:"NOT NULL"`
+	RepositoryID int64     `xorm:"NOT NULL"`
```

Comment:
Adding an `index` xorm tag to `RepositoryID` and an unique index to `Oid` and `RepositoryID` is better.